.github/workflows/renovate.yml

```diff
@@ -0,0 +1,25 @@
+name: Renovate
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: "Override default log level"
+        required: false
+        default: "info"
+        type: string
+      overrideSchedule:
+        description: "Override all schedules"
+        required: false
+        default: "false"
+        type: string
+  # Run twice in the early morning (UTC) for initial and follow up steps (create pull request and merge)
+  schedule:
+    - cron: '30 4,6 * * *'
+
```

Comment:
Should it have explicit permissions set as in [rancher/rancher](https://github.com/rancher/rancher/blob/70bd7e78a83d409eabacbbd5a5183a91f25ce58d/.github/workflows/renovate-vault.yml#L19-L21)?